src/styles/main.scss

```diff
@@ -36,7 +36,7 @@ a {
 }
 
 .card__layout {
-  width: 944px;
+  width: $card-width * 4 + $card-margin-right * 3;
```

Comment:
use max-width because if you have a smaller screen, everything breaks

Reply:
+